src/builder-manifest.c

```diff
@@ -2834,7 +2834,16 @@ builder_manifest_finish (BuilderManifest *self,
 
       if (self->build_runtime)
         {
-          debuginfo_dir = g_file_resolve_relative_path (app_dir, "usr/lib/debug");
+          const char * libdir = builder_options_get_libdir (self->build_options, context);
```

Comment:
I don't think we want to modify /usr/lib/debug actually. It already contains the full path below it. For example `/usr/lib/debug/usr/lib/libglib-2.0.so.0.5200.3.debug`.